app/build.gradle

```diff
@@ -4,6 +4,8 @@ apply plugin: 'io.sentry.android.gradle'
 apply plugin: 'com.noveogroup.android.check'
 apply from: './jacoco.gradle'
 
+def app_name = System.getenv('app_name') ?: "eventyay organizer"
```

Comment:
Environment variables should generally be upper cased. Dunno how I missed that in initial review

Reply:
yeah. Initially, I tried APP_NAME but that is always returning `Gradle` no matter what is set. you may suggest another name. but app_name or APP_NAME is convenient.

Reply:
Ok :+1: